redis/src/cmd.rs

```diff
@@ -353,6 +353,48 @@ impl RedisWrite for Cmd {
 
         CmdBufferedArgGuard(self)
     }
+
+    #[cfg(feature = "bytes")]
+    fn bufmut_for_next_arg(&mut self, capacity: usize) -> impl bytes::BufMut + '_ {
+        self.data.reserve(capacity);
+        struct CmdBufferedArgGuard<'a>(&'a mut Cmd);
+        impl Drop for CmdBufferedArgGuard<'_> {
+            fn drop(&mut self) {
+                self.0.args.push(Arg::Simple(self.0.data.len()));
```

Comment:
what happens if the guard is dropped without adding any argument?

Reply:
It works the same way as `writer_for_next_arg`, see https://github.com/redis-rs/redis-rs/pull/905#discussion_r1947894825

Reply:
And that's just from February. I'm never going to learn 🤦 
Yes, please also add a clarification to all relevant docs.